operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -850,6 +850,12 @@ ingester:
     # Useful if the volume's root directory is not empty
     subPath: ''
 
+    # -- Enable StatefulSetAutoDeletePVC feature
+    # https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#persistentvolumeclaim-retention
+    enableStatefulSetAutoDeletePVC: false
```

Comment:
what's the need for having this flag if we are also checking the kubernetes version?

Reply:
i see it was added to loki in 2022 with the same, but that PR doesn't have any notes on why this is necessary

Reply:
It's just in case you do not wish to configure this retention policy for Kubernetes 1.27 or higher